### ISSUE TITLE
Readme Section 3

### ISSUE THREAD

**Issue by wobbol** [cjdelisle/cjdns#720]
I guess I will just make another list...:

* JSON
  + commas are required at the end of all curly brace statements when,  
     another curly brace statement follows it.
  + should be encouraged to put extra identifying information in extra key/value pairs.
    * the admin interface returns two functions that accept a user value named  
      AuthorizedPasswords_add() AuthorizedPasswords_remove() but there is not a lick of   
      documentation.   
      Does this mean that if the user puts `"user" : "JonDoe",` on the line above one of  
      their "password" JSON keys this line will get parsed and will be accessible through the admin interface?
  + This is on the same note ad above but, it should be stated outright if adding unrecognized  
      key/value pairs will "mess anything up"
* Says nothing about adding connections through the admin interface.  

This one is more taste than anything else but,
puts too much emphasis on making the user know the _name_ of the thing they are doing and not enough on _what_ they need to do to get up and running.

Sections devoted to making config files need to be directive and have a clear path of action. I could not follow this section when I tried to connect to hype and many others (alright only 3) have had the same trouble. It is extremely discouraging especially for users that had a hard time building cjdroute to begin with to be met with such a terse section.  
**tl;dr** please direct users to the documentation project.

**Comment by ansuz** [comment]
[Lars](//github.com/lgierth) has been making an effort to port parts of the [documentation project](//github.com/projectmeshnet/documentation) into [the main cjdns repo](//github.com/cjdelisle/cjdns/tree/master/doc) once they have been properly scrutinized by active members of [Project Meshnet](//github.com/ProjectMeshnet). 

With regards to the conf, I recommend reading [this report I wrote today](//github.com/ProjectMeshnet/documentation/blob/master/bugs/connectTo-overflow.md) regarding [issue 701](//github.com/cjdelisle/cjdns/issues/701).

There will be a major review and restructuring of this documentation (where necessary) this weekend. As such, we should consider closing this issue.

**Comment by ansuz** [comment]
In the future, please consider submitting requests for documentation to the documentation project, as this issue will have to be closed by cjd himself.
